ls1intum/Artemis
# [Bugfix] - Show number of teams and team participations

### Checklist

### Motivation and Context


We currently display the number of student participations and number of students as max also for team exercises. This does not make that much sense since participations are done in teams.

### Description

We count the team participations in correlation to the maximum number of teams in this exercise

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate into the exercise details of an team exercise (or create a new team exercise)
4. Check that when no teams are added yet, the participationrate says 0 / 0
5. Add some teams to this exercise
6. Check that the denominator of the participation rate doughnut chart is the number of teams in the exercise
7. Participate and submit with a team and check that the number of participations in the doughnut chart in this exercise increases by 1


### Screenshots


We currently display the number of students in the course:
<img width="955" alt="Screenshot 2021-06-12 at 11 06 33" src="https://user-images.githubusercontent.com/44401560/121771456-c44b2700-cb6f-11eb-9766-7094fa463174.png">



Now, with two teams created for the exercise:
<img width="957" alt="Screenshot 2021-06-12 at 11 10 26" src="https://user-images.githubusercontent.com/44401560/121771315-f6a85480-cb6e-11eb-8594-4eb82d403079.png">
